src/components/Questionnaire/QuestionnaireResultsModal.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import {DefaultButton as Button} from "@common/components";
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepButton from '@mui/material/StepButton';
+import Modal from '@mui/material/Modal';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import {Stack} from "@mui/material";
+import {useTranslation} from "react-i18next";
+
+//Can not shorten import, tests fail to recognize i18n.use...
+import {GraphListK} from "./GraphListK";
+import {TableListK} from "./TableListK";
+import {ResultDescILS} from "./ResultDescriptionILS";
+import {ResultDescListK} from "./ResultDescriptionListK";
+import {GraphILS} from "./GraphILS";
+import {TableILS} from './TableILS';
+
+const styleButtonClose = {
+    position: 'sticky',
+    left: '99%',
+    top: '1%',
+    p: 2,
+}
+
+const styleBox = {
+    position: 'absolute',
+    left: '8%',
+    right: '8%',
+    top: '5%',
+    overflow: 'auto',
+    maxHeight: '83%',
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 1,
+};
+
+export const QuestionnaireResultsModal = () => {
+    const {t} = useTranslation();
+
+    const steps = [t("components.QuestionnaireResults.ResultDescriptionILS.ILSResults"), t("components.QuestionnaireResults.ResultDescriptionILS.ListKResults")];
+    const [open, setOpen] = React.useState(false);
```

Comment:
Lieber useState direct importieren als * from react```suggestion
    const [open, setOpen] = useState(false);
```